p2p/src/net/types/services.rs

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2023 RBB S.r.l
+// opensource@mintlayer.org
+// SPDX-License-Identifier: MIT
+// Licensed under the MIT License;
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://github.com/mintlayer/mintlayer-core/blob/master/LICENSE
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use serialization::{Decode, Encode};
+
+#[derive(Eq, PartialEq, Clone, Copy, Debug, Encode, Decode, Default)]
+pub struct Service(u32);
+
+impl Service {
+    /// Transactions
+    pub const TRANSACTIONS: Service = Service(1 << 0);
+
+    /// Blocks
+    pub const BLOCKS: Service = Service(1 << 1);
+
+    /// Peer address announcements from new nodes joining the network
+    pub const PEER_ADDRESSES: Service = Service(1 << 2);
+}
+
+#[derive(Eq, PartialEq, Clone, Copy, Debug, Encode, Decode, Default)]
```

Comment:
There's a problem with this being default... what does it mean? It's meaningless and is even invalid. What is a node that has zero services?!

Reply:
Right, removed. Thanks.